molo/profiles/templates/profiles/editprofile.html

```diff
@@ -2,15 +2,16 @@
 {% load i18n %}
 
 {% block content %}
-<div class="profile-style clearfix">
-	<form method="post" id="frmProfileEdit" action="" class="profile">
-	{% csrf_token %}
-	  {% trans "Edit your Profile" %}
-	  <table border="0">
-	            {{ form.as_table }}
-	  </table>
-	  <input type="submit" value='{% trans "Update profile" %}' class="btn btn-wide">
-	</form>
+<div class="signin-form">
+	<div class="block turq">
+		<form method="post" id="frmProfileEdit" action="{% url 'molo.profiles:edit_my_profile' %}">{% csrf_token %}
```

Comment:
This form (and TuneMe) is missing the `date of birth` field, meaning a user can never change it, which is bad..